bottlerocket/samples/Makefile.toml

```diff
@@ -133,6 +140,20 @@ if [ -n "${MGMT_CLUSTER_KUBECONFIG_PATH}" ]; then
 fi
 ''']}
 
+[tasks.hardware-csv-base64.env]
+HARDWARE_CSV_BASE64 = {script=['''
+if [ -n "${HARDWARE_CSV_PATH}" ]; then
+    cat ${HARDWARE_CSV_PATH} | base64 -w 0
```

Comment:
Not all versions of `base64` support the `-w` flag and instead have word wrapping by default. It's not the prettiest, but you might want to implement something similar to:
```
if base64 -w0 <<< "" &>/dev/null; then
    cat ${HARDWARE_CSV_PATH} | base64 -w0
else
    cat ${HARDWARE_CSV_PATH} | base64
fi
  ```

Reply:
`echo "" | base64 -w0 >/dev/null 2>&1` might be even more compatible.